verification/verify_test.go

```diff
@@ -57,19 +61,42 @@ func TestVerify(t *testing.T) {
 	assert.Nil(res)
 	assert.Equal(ErrPixelsAbsent, err)
 
-	// check pixel count fails
+	// check pixel count fails w/ external verifier
 	data.Segments = append(data.Segments, &net.TranscodedSegmentData{Url: "def", Pixels: verifier.results.Pixels[1]})
 	assert.Len(data.Segments, len(verifier.results.Pixels)) // sanity check
 	res, err = sv.Verify(&Params{Results: data})
 	assert.Nil(res)
 	assert.Equal(ErrPixelMismatch, err)
 
-	// Check pixel count succeeds
+	// Check pixel count succeeds w/ external verifier
 	data.Segments[0].Pixels = verifier.results.Pixels[0]
 	res, err = sv.Verify(&Params{Results: data})
 	assert.Nil(err)
 	assert.NotNil(res)
 
+	// check pixel count fails w/o external verifier
+	sv = NewSegmentVerifier(&Policy{Verifier: nil, Retries: 2}) // reset
+	data = &net.TranscodeData{Segments: []*net.TranscodedSegmentData{
+		{Url: "abc", Pixels: verifier.results.Pixels[0] + 1},
+	}}
+	data.Segments = append(data.Segments, &net.TranscodedSegmentData{Url: "def", Pixels: verifier.results.Pixels[1]})
+	assert.Len(data.Segments, len(verifier.results.Pixels)) // sanity check
+	res, err = sv.Verify(&Params{Results: data, Orchestrator: &net.OrchestratorInfo{TicketParams: &net.TicketParams{}}})
```

Comment:
Looks like there is still an extra `TicketParams` here.

Reply:
fixed in 6c66a847fbebaeb3eb0c0f5bb1aafd170d31bf4e